src/site/stages/build/process-cms-exports/schemas/output/node-event_listing.js

```diff
@@ -10,7 +10,7 @@ const reverseFieldSchema = {
         entity: partialSchema(eventSchema, [
           'title',
           'entityUrl',
-          'fieldDate',
+          'fieldDatetimeRangeTimezone',
```

Comment:
Just confirming, the field `fieldDate` didn't change across the whole CMS export did it?

Reply:
Well it will be null going forward for all new events, in both cms export and graphql.

I put the field back per @theodur 's request.